packages/web-pkg/src/components/SearchBarFilter.vue

```diff
@@ -64,12 +64,12 @@ export default defineComponent({
     const locationOptions = ref<LocationOption[]>([
       {
         id: SearchLocationFilterConstants.currentFolder,
-        title: $gettext('Current Folder'),
+        title: 'Current Folder',
```

Comment:
Please don't remove `$gettext` here. This method (and its variations) are an indicator for our language string extractor to sync strings to Transifex, where they eventually get translated.

Reply:
Right, looks like my previous approach was better. And it seems that moving "currentFolderEnabled" prop from computed into locationOptions (as computed) fixed the issue